express-generator-typescript/lib/auth-proj/spec/index.ts

```diff
@@ -43,6 +43,7 @@ jasmine.onComplete((passed: boolean) => {
     } else {
         logger.err('At least one test has failed :(');
     }
+    jasmine.exitCodeCompletion(passed);

```

Comment:
In my testing, an explicit call was needed to `exitCodeCompletion` to force jasmine to exit at this stage after running all tests.

Reply:
Thank you for this, this should be added!!!!